nativelib/src/main/resources/os_win_random.cpp

```diff
@@ -0,0 +1,145 @@
+#include <stdio.h>
+
+extern "C" {
+    // we need to create fake uses of functions (snprintf etc) otherwise it will be stripped from the file
+    int scalanative_snprintf( char * s, size_t n, const char * format, va_list args)
+    {
+        return snprintf(s, n, format, args);
+    }
+
+    int scalanative_sprintf( char * s, const char * format, va_list args)
+    {
+        return sprintf(s, format, args);
+    }
+
+#ifdef _WIN32
+    /*
+    #include <random>
+    std::default_random_engine generator;
+    std::uniform_real_distribution<double> distr(0.0,1.0);
+    double erand48(int X)
+    {
+        return distr(generator);
+    }
+    */
+    
+    #include <stdlib.h>
+    #include <math.h>
+
+    #define RAND48_SEED_0   (0x330e)
+    #define RAND48_SEED_1   (0xabcd)
+    #define RAND48_SEED_2   (0x1234)
+    #define RAND48_MULT_0   (0xe66d)
+    #define RAND48_MULT_1   (0xdeec)
+    #define RAND48_MULT_2   (0x0005)
+    #define RAND48_ADD      (0x000b)
+
+    unsigned short _rand48_seed[3] = {
+        RAND48_SEED_0,
+        RAND48_SEED_1,
+        RAND48_SEED_2
+    };
+    unsigned short _rand48_mult[3] = {
+        RAND48_MULT_0,
+        RAND48_MULT_1,
+        RAND48_MULT_2
+    };
+    unsigned short _rand48_add = RAND48_ADD;
+
+    void _dorand48(unsigned short xseed[3]) 
```

Comment:
This function was probably taken from https://gist.github.com/mortennobel/8665258. You should add some attribution. Did you make sure that the license complies with the CLA?